Header header
int16 polynomial_degree
int16 half_datalength
float64[] timelist
float64[] coefflist